src/Containers.jsx

```diff
@@ -41,22 +41,22 @@ import PruneUnusedContainersModal from './PruneUnusedContainersModal.jsx';
 
 const _ = cockpit.gettext;
 
-const ContainerActions = ({ container, containerDetail, healthcheck, onAddNotification, version, localImages, updateContainer }) => {
+const ContainerActions = ({ container, healthcheck, onAddNotification, version, localImages, updateContainer }) => {
     const Dialogs = useDialogs();
     const [isActionsKebabOpen, setActionsKebabOpen] = useState(false);
-    const isRunning = container.State == "running";
-    const isPaused = container.State === "paused";
+    const isRunning = container.State.Status == "running";
+    const isPaused = container.State.Status === "paused";
 
     const deleteContainer = (event) => {
         setActionsKebabOpen(false);
 
-        if (container.State == "running") {
+        if (container.State.Status == "running") {
             const handleForceRemoveContainer = () => {
                 const id = container ? container.Id : "";
 
                 return client.delContainer(container.isSystem, id, true)
                         .catch(ex => {
-                            const error = cockpit.format(_("Failed to force remove container $0"), container.Names);
+                            const error = cockpit.format(_("Failed to force remove container $0"), container.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)